web/src/hotkey.js

```diff
@@ -990,6 +996,13 @@ export function process_hotkey(e, hotkey) {
         case "mark_unread":
             unread_ops.mark_as_unread_from_here(msg.id);
             return true;
+        case "view_read_receipts_or_view_selected_stream":
+            // Show read receipts for the selected message
+            // if the stream settings overlay is not open.
+            if (!overlays.streams_open()) {
```

Comment:
Do we need this check? I would have expected this block to already be after the point where we checked whether any overlays were open and exited if so.